locales/base/translation.json

```diff
@@ -1366,5 +1366,11 @@
     "swapTo": "To",
     "insufficientFunds": "Insufficient {{token}} balance. Try a smaller amount or choose a different token.",
     "fetchSwapQuoteFailed": "Sorry for the delay! It's taking longer than usual to get the best exchange rate. Please try again in a few minutes."
+  },
+  "swapContent": {
```

Comment:
`swapTransactionItem`? having more specific translation key names (that kind of describe where in the app the key is used) helps our translators to translate more accurately

Reply:
I changed it to `swapTransactionDetailPage`